Modules/BenchPress.Azure/Tests/Public/Confirm-Resource.Tests.ps1

```diff
@@ -17,7 +17,7 @@ Describe "Confirm-Resource" {
     }
 
     It "Calls Get-ResourceByType; returns true when Get-ResourceByType returns a Success ConfirmResult." {
-      Mock Get-ResourceByType{ $ConfirmResult }
+      Mock Get-ResourceByType{ $ConfirmResult } -Verifiable -RemoveParameterType 'ResourceType'
```

Comment:
Curious: what does adding `-RemoveParameterType 'ResourceType'` do?

Reply:
Pester could not mock the enum I introduced, so it kept throwing an error when the mocked function was invoked. But fortunately they have a parameter for this case. You can see the documentation [here](https://pester.dev/docs/commands/mock/#-removeparametertype)